app/models/field.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Field < ApplicationRecord
+  enum data_type: {
+    short_text: 0,
+    long_text: 1
+  }
```

Comment:
```suggestion
  enum data_type: {
    short_text: 0,
    long_text: 1
  }, validate: true
```
With this option, Rails will validate the presence and the inclusion of correct values in the enum. So, instead of throwing an `ArgumentError`, the enum will throw an `ActiveRecord::RecordInvalid` error when an invalid value is assigned to `data_type`.

Reply:
I tried this approach, but it's didn't work. I searched for other ways to do the validation but none works, each one returns a different error. We can have a meeting to try to resolve this if you can.